Tests/RespondersTests/RouteResponderTest.swift

```diff
@@ -98,7 +98,7 @@ class RouteResponderTest: XCTestCase {
 
       let expected = "GET /someRoute HTTP/1.1"
 
-      XCTAssertEqual(response.body!, expected.toBytes)
+      XCTAssertEqual(response.body!.toBytes, expected.toBytes)
```

Comment:
This diff is because of a recent change I made to represent a response body property as `BytesRepresentable` instead of `[UInt8]`